### PR TITLE
Add delete comment functionality to back end

### PR DESCRIPTION
**Pull request by max-friedman**

### CODE REVIEW

File: project/src/main/java/com/google/sps/Models/Database.java [hunk 1]
```diff
@@ -412,11 +412,27 @@ private static ArrayList getListProperty(Entity entity, String prop) {
...
+    ArrayList<Long> commentIDs;
+    if(document.getCommentIDs() != null) {
+      commentIDs = document.getCommentIDs();
+    } else {
+      return new ArrayList<Comment>();
+    }        
+
     ArrayList<Comment> commentObjects = new ArrayList<Comment>();
     for(Long c: commentIDs) {
       commentObjects.add(getCommentbyID(c));
```
Comment (g-luo) [inline project/src/main/java/com/google/sps/Models/Database.java:424]: nit: use consistent camel case (getCommentByID)

**Review by g-luo — APPROVED** [review]
lgtm

File: project/src/main/java/com/google/sps/Servlets/DeleteCommentServlet.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.gson.*;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.ArrayList;
```
Comment (tin0d0) [inline project/src/main/java/com/google/sps/Servlets/DeleteCommentServlet.java]: dont need arraylist, date, simpledateformat, multipartconfig, collectors, userservice, and userservice factory imports.

**Review by tin0d0 — APPROVED** [review]
LGTM

**Review by HidyHan — APPROVED** [review]

**Review by frankpzn — APPROVED** [review]

**Pull request merged**